workspaces/global-header/plugins/global-header/src/components/HeaderDropdownComponent/ApplicationLauncherDropdown.tsx

```diff
@@ -0,0 +1,124 @@
+
+export const ApplicationLauncherDropdown = () => {
+  const { anchorEl, handleOpen, handleClose } = useDropdownManager();
+
+  const mountPoints = useApplicationLauncherDropdownMountPoints();
+
+  const sectionsObject = useMemo(() => {
+    const groupedSections: Record<
+      string,
+      { sectionLink?: string; sectionLinkLabel?: string; items: any[] }
+    > = {};
+
+    (mountPoints ?? []).forEach(mp => {
+      const section = mp.config?.section ?? '';
+      const sectionLink = mp.config?.sectionLink ?? '';
+      const sectionLinkLabel = mp.config?.sectionLinkLabel ?? '';
+
+      if (!groupedSections[section]) {
+        groupedSections[section] = { sectionLink, sectionLinkLabel, items: [] };
+      }
+      groupedSections[section].items.push({
+        Component: mp.Component,
+        icon: mp.config?.props?.icon,
+        label: mp.config?.props?.title,
+        link: mp.config?.props?.link,
+        external: mp.config?.props?.external ?? false,
+        priority: mp.config?.priority ?? 0,
+      });
+    });
+
+    Object.values(groupedSections).forEach(section => {
+      section.items.sort((a, b) => (b.priority ?? 0) - (a.priority ?? 0));
+    });
+
+    return groupedSections;
+  }, [mountPoints]);
+
+  const sections = Object.entries(sectionsObject);
+
+  const NoApplicationLinks = () => {
+    return (
+      <InfoCard>
+        <Box
+          display="flex"
+          flexDirection="column"
+          alignItems="center"
+          justifyContent="center"
+          textAlign="center"
+          py={4}
+          px={2} // Added padding to control width
+          maxWidth={300} // Set max width to constrain text expansion
+          mx="auto"
+        >
+          <AppRegistrationIcon sx={{ fontSize: 64 }} color="disabled" />
+          <Typography variant="h6" sx={{ mt: 2, color: 'text.primary' }}>
+            No application links configured
+          </Typography>
+          <Typography
+            variant="body2"
+            sx={{ mt: 1, color: 'text.secondary', maxWidth: 250 }}
+          >
+            Configure application links in dynamic plugin config for quick
+            access from here.
+          </Typography>
+        </Box>
+      </InfoCard>
+    );
+  };
+
+  return (
+    <HeaderDropdownComponent
+      buttonContent={<AppsIcon />}
+      tooltip="Application Launcher"
+      isIconButton
+      onOpen={handleOpen}
+      onClose={handleClose}
+      anchorEl={anchorEl}
+    >
+      {sections.length > 0 ? (
+        sections.map(
+          ([section, { sectionLink, sectionLinkLabel, items }], index) => (
+            <MenuSection
+              key={section}
+              sectionLabel={section}
+              optionalLink={sectionLink}
+              optionalLinkLabel={sectionLinkLabel}
+              items={items}
+              handleClose={handleClose}
+              hideDivider={index === sections.length - 1}
```

Comment:
in your screenshot it looks like the final divider isn't properly hidden:

![image](https://github.com/user-attachments/assets/f8aec5d3-c766-42a3-a66c-5d03501e318b)

can you double check whether this is due to a misconfiguration in the `app-config`, or if this is a bug?